.github/workflows/publish-major-minor.yml

```diff
@@ -0,0 +1,33 @@
+name: "Publish Major-Minor-Tags"
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  push-tags:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install packaging
+        run: pip install -U packaging
+      - name: Get versions
+        id: get_versions
+        run: python .github/workflows/publish-major-minor.py ${{ github.ref }}
+      - name: Push Tags Version
+        if: steps.get_versions.outputs.original_tag_name != ''
```

Comment:
What is this condition for?

Reply:
This checks if the code in the if condition in the python script did run or not, since only then the output is set.